style.css

```
@@ -127,3 +127,273 @@ img {
     gap: 20px;
   }
 }
+
+/* about */
+.about {
+  background-color: var(--color-background-black);
+  display: flex;
+  justify-content: center;
+}
+
+.about-container {
+  max-width: 880px;
+  padding: 80px 40px;
+}
+
+.about-main-copy {
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  letter-spacing: 0.02em;
+  color: var(--color-white);
+  margin: 0;
+  padding-bottom: 1em;
+  text-align: center;
+}
+
+.about-main-copy-accent-red {
+  color: var(--color-accent-red);
+}
+
+.about-main-copy-accent-yellow {
+  color: var(--color-accent-yellow);
+}
+
+.about-content {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  column-gap: 20px;
+}
+
+.about-text {
+  font-family: var(--font-family-body);
+  color: var(--color-white);
+  font-size: 16px;
+  font-weight: 400;
+  margin: 0;
+}
+
+.about-image-box {
+  width: 320px;
+  height: 180px;
+}
+
+.about-image {
+  width: 320px;
+  height: 180px;
+  object-fit: cover;
+  padding: 0;
+  margin: 0;
+}
+
+@media (max-width: 750px) {
+  .about-container {
+    padding: 40px 20px;
+  }
+  .about-image-box {
+    display: none;
+  }
+  .about-main-copy {
+    font-size: 24px;
+    letter-spacing: normal;
+    padding-bottom: 40px;
+  }
+  .about-content {
+    gap: 0;
+  }
+  .about-text > br {
+    display: none;
+  }
+}
+
+/* courses */
+.courses {
+  background-color: var(--color-background-navy);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 80px 0;
+}
+
+.courses-container {
+  max-width: 880px;
+  padding: 0 40px;
+}
+
+.courses-main-copy {
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  letter-spacing: 0.02em;
+  color: var(--color-white);
+  margin: 0;
+  text-align: center;
+  margin-bottom: 40px;
+  font-weight: 400;
+}
+
+.courses-card {
+  display: flex;
+  margin-bottom: 20px;
+  filter: drop-shadow(4px 4px 0 var(--color-accent-red));
+  height: 150px;
+  background-color: var(--color-accent-yellow);
+}
+
+.courses-card:hover {
+  filter: drop-shadow(10px 10px 0 var(--color-accent-red));
+  transition: 0.3s;
+  background-color: var(--color-course-hover-yellow);
+}
```

```suggestion
.courses-card:hover {
  filter: drop-shadow(10px 10px 0 var(--color-accent-red));
  background-color: var(--color-course-hover-yellow);
}
```
hover時に、ピンク色の部分が[お手本のサイト](https://shinonome.io/frontend-basic-2023-html-css-assignment-baseline/chapter2/)と異なっているので修正をお願いします。
.courses-cardの方に、`transition-duration: 0.3s;`を記述してみてください！

---

修正しました！